Content.Server/server_config.toml

```diff
@@ -0,0 +1,158 @@
+# Welcome to the example configuration file for Space Station 14!
+# If you are in a watchdog enviroment, please create config.toml in your instence's root directory instead of editing this.
+# If you are in a development environment, ensure you are editing /bin/Content.Server/server_config.toml. Be aware that this file may be overwritten on build.
+
+# This configuration file contains **some popular** options most often changed by sandboxers and server hosts.
+# Along with a description of what they do. Look at the end of the file for a list of all options available if it's not listed here.
+
+[net]
+# The ticks per second the server runs at.
+# Lower values mean less load on the server, but also less responsiveness.
+# Higher values mean more load, but also more responsiveness. 
+# 30 is what most servers on the hub use and going higher has minimal effect.
+tickrate = 30
+# The port the server will bind to. 1212 is the default port for Space Station 14.
+port = 1212
+# Attempt to automatically forward the port using UPnP. Change to true to enable.
+# If it doesn't work, please follow the instructions here: https://docs.spacestation14.com/en/server-hosting/port-forwarding.html
+upnp = false
+# The maximum number of players that can connect to the server. (Including admins)
+# Look at the game section for the soft max players setting. Which is what is displayed in the server list.
+max_connections = 256
+
+[game]
+# Name of the server. This is what will be displayed in the server list and lobby.
+hostname = "MyServer"
+# Decscription of the server. This is what will be displayed in the server list.
+desc = "Just another server, don't mind me!"
+# The maximum number of players that can connect to the server. (Exluding admins)
+soft_max_players = 30
+# Controls the lobby timer in seconds.
+lobbyduration = 150
+# Should roles be restricted based on time?
+role_timers = true
+# Check connecting player ip against the IPIntel API (aka stop VPN/Proxy connections). Useful for hosts.
+# Read more here https://getipintel.net/
+ipintel_enabled = false
+# Required contact email to use IPIntel.
+ipintel_contact_email = ""
+# IPIntel score to reject connections.
+ipintel_bad_rating = 0.95
+
+[ic]
+# Allows visibility and editing of flavor text (character descriptions)
+flavor_text = false
+
+[admin]
+# Should players be able to see their own notes?
+see_own_notes = false
+
+[server]
+# Change this to have the changelog and rules "last seen" date stored separately from other servers.
+id = "unknown_server_id"
+
+[events]
+# Controls if the game should run station events. (Meteors, Sleeper Syndicates ETC)
+eneabled = true
```

Comment:
eneabled

You should really make a test that makes sure this config can be parsed.